pubnub-core/pubnub-core-api/src/main/kotlin/com/pubnub/api/retry/RetryConfiguration.kt

```diff
@@ -65,21 +70,37 @@ sealed class RetryConfiguration {
      * @property excludedOperations List of [RetryableEndpointGroup] to be excluded from retry.
      */
     class Exponential internal constructor(
-        var minDelayInSec: Duration = MIN_DELAY.seconds, // min value is 2
```

Comment:
This is also strange to me. I remove this comment ( // min value is 2) because running
` ./gradlew ktlintCheck --rerun-tasks`
or
` ./gradlew clean build`
gives error like:
"A comment in a 'value_parameter_list' is only allowed when placed on a separate line (cannot be auto-corrected) (standard:discouraged-comment-location)
"
but at that time it didn't pointed out that the same issue is 38, 39 ,40 so the build passed successfully.

Then I did some other changes and getting error for 38, 39,40 lines :|

Reply:
either disable the ktlint rule, or move the comment above the line like so:
```
// min value is 2
var minDelayInSec: Duration = MIN_DELAY.seconds, 
```

it's possible that these lines were part of the generated baseline to ignore this error, but after editing the line numbers changed and the baseline file no longer matched